docs/install.md

```diff
@@ -89,31 +89,36 @@ Linux versions).
 
 ## Installing via Docker
 
-Option 1 (Basic, 3.13GB):
+Option 1 (3.13GB, 1.14GB Compressed): Base Image without JupyterLab 
+
+_Add --compress for a smaller image size_
 
 ```bash
 docker build --target base -t openspiel . --rm
 ```
 
-Option 2 (Slim, 2.26GB):
+```bash
+docker run -it --entrypoint /bin/bash openspiel
+```
 
 ```bash
-docker build --target python-slim -t openspiel . --rm
+docker run openspiel python3 python/examples/matrix_game_example.py
+docker run openspiel python3 python/examples/example.py
 ```
 
-If you are only interested in developing in Python, use the second image. You
-can navigate through the runtime of the container (after the build step) with:
+Option 2: Base image with JupyterLab (_very useful for a 'Google Colab' experience_)
```

Comment:
E.g. I'd like it if we preserved the original option 2, and added this as option 3.

Reply:
Hello @finbarrtimbers, the Dockerfile always works _incrementally_ and JupyterLab needs Ubuntu to operate. So in order to also support both JupyterLab and a slim image without JupyterLab, I must generate another Dockerfile if that is desired instead?

Reply:
Yes, I'd like it if you did that!